x-pack/legacy/plugins/infra/public/containers/logs/log_summary/bucket_size.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useMemo } from 'react';
+
+export function useBucketSize(
+  startTimestamp: number | null,
+  endTimestamp: number | null
+): number | null {
+  const bucketSize = useMemo(() => {
```

Comment:
What's the purpose of making this into a hook? This seems to be behaving like an ordinary function.

Reply:
Initially I had the body of this hook (`useMemo(() => ...)` ) as part of the `useLogSummary` hook. Later I had to use the same `useMemo` in the `useLogSummaryHighlights` hook, so I extracted it as a hook in itself.

To be honest, maybe the `useMemo` is overkill and we can indeed make this into a normal function, and not cache the value (it's a fast calculation anyway).